snapshots/Integration_ALM_Multi.json

```diff
@@ -1,3 +1,3 @@
 {
-  "gasUsed": "47579"
+  "gasUsed": "137087"
```

Comment:
What is this file? Can we gitignore snapshots, or figure out what's generating this?

(is it useful/used somewhere?)